client/src/components/RelatedAndComparison/assets/Outfit.jsx

```diff
@@ -4,43 +4,46 @@ import React, { useState } from 'react';
 import { FaArrowAltCircleLeft, FaArrowAltCircleRight } from 'react-icons/fa';
 import Images from './Images.jsx';
 
-const Outfit = ({ yourOutfit, handleClick }) => {
+const Outfit = ({ yourOutfit, handleClick, openModal }) => {
   const [currentCard, setCurrentCard] = useState(0);
-  const length = yourOutfit.length;
+  const length = yourOutfit.length - 5;
```

Comment:
It seems like this is allowing you to make sure that when there are no items left to render, it renders the first 5 cards again. One idea would be to have the last 4 show and the first product show to the right. If you don't want to implement it that way. I would just make the right arrow do nothing if it is at the max length. That is, I would change line 15 from
? 0 to ? currentCard.

Reply:
Smart! I was going to implement that later, but removing the arrow for the time being is a good idea.

Reply:
Another thing to keep in mind is what if your length is actually less than 5?